README.md

```diff
@@ -172,6 +172,7 @@
 * **[LMCache](https://github.com/LMCache/LMCache)**: 10x Faster Long-Context LLM By Smart KV Cache Optimizations. ![Stars](https://img.shields.io/github/stars/lmcache/lmcache.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/lmcache/lmcache?color=green) ![LastCommit](https://img.shields.io/github/last-commit/lmcache/lmcache?color=green) ![Tag](https://img.shields.io/badge/kvcache-orange)
 * **[Mooncake](https://github.com/kvcache-ai/Mooncake)**: Mooncake is the serving platform for Kimi, a leading LLM service provided by Moonshot AI. ![Stars](https://img.shields.io/github/stars/kvcache-ai/mooncake.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/kvcache-ai/mooncake?color=green) ![LastCommit](https://img.shields.io/github/last-commit/kvcache-ai/mooncake?color=green)
 * **[OpenLLM](https://github.com/bentoml/OpenLLM)**: Run any open-source LLMs, such as DeepSeek and Llama, as OpenAI compatible API endpoint in the cloud. ![Stars](https://img.shields.io/github/stars/bentoml/openllm.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/bentoml/openllm?color=green) ![LastCommit](https://img.shields.io/github/last-commit/bentoml/openllm?color=green)
+* **[Kaito](https://github.com/kaito-project/Kaito)**: Kubernetes operator for large-model inference and fine-tuning, with GPU auto-provisioning, container-based hosting, and CRD-based orchestration. ![Stars](https://img.shields.io/github/stars/kaito-project/Kaito.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/kaito-project/Kaito?color=green) ![LastCommit](https://img.shields.io/github/last-commit/kaito-project/Kaito?color=green)
```

Comment:
We're alphabetized, could you reorder this project? Thanks!

Reply:
No worries, I've updated accordingly, thank you!